CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 5.2.1 (Aug 19, 2025)
 
+FEATURES:
```

Comment:
Can we move this above `## 5.2.1 (Aug 19, 2025)` under `Unreleased`?
And maybe update it to say something along the lines of:

Add support for write-only token argument in `vault_terraform_cloud_secret_backend` resource ([#2483](https://github.com/hashicorp/terraform-provider-vault/pull/2483))

That way we stay consistent with [other feature changelog entries](https://github.com/hashicorp/terraform-provider-vault/blob/47c7a435fddf60bd3fb6f84340a95b9a52ca874d/CHANGELOG.md?plain=1#L21-L25)

Reply:
Small nit: Would you mind re-wording the changelog entry so it stays consistent with others? Thank you 🙏